tools/depends/target/ffmpeg/Makefile

```diff
@@ -2,81 +2,34 @@ include ../../Makefile.include
 include FFMPEG-VERSION ../../download-files.include
 DEPS = ../../Makefile.include FFMPEG-VERSION Makefile ../../download-files.include
 
-export PKG_CONFIG_LIBDIR=$(PREFIX)/lib/pkgconfig
-
-# configuration settings
-ffmpg_config = --prefix=$(PREFIX) --extra-version="kodi-$(VERSION)"
-ffmpg_config += --cc="$(CC)" --cxx="$(CXX)" --ar=$(AR) --ranlib=$(RANLIB) --strip=$(STRIP)
-ffmpg_config += --extra-cflags="$(CFLAGS)"
-ffmpg_config += --extra-cxxflags="$(CXXFLAGS)"
-ffmpg_config += --extra-ldflags="$(LDFLAGS)"
-ffmpg_config += --pkg-config=$(NATIVEPREFIX)/bin/pkg-config
+BUILD_TYPE = Release
+ifeq ($(DEBUG_BUILD),yes)
+  BUILD_TYPE = Debug
+endif
 
-ffmpg_config += --disable-doc
-ffmpg_config += --disable-devices
-ffmpg_config += --disable-programs
-ffmpg_config += --disable-sdl2
-ffmpg_config += --enable-gpl
-ffmpg_config += --enable-postproc
-ffmpg_config += --enable-runtime-cpudetect
-ffmpg_config += --enable-pthreads
-ffmpg_config += --enable-gnutls
-ffmpg_config += --enable-libdav1d
-ffmpg_config += $(FFMPEG_CONFIGURE_OPTIONS)
+CMAKE_ARGS=-DCMAKE_BUILD_TYPE=$(BUILD_TYPE) \
+           -DCMAKE_MODULE_PATH=$(CMAKE_SOURCE_DIR)/cmake/modules \
+           -DPKG_CONFIG_PATH=$(PREFIX)/lib/pkgconfig \
+           -DCMAKE_INSTALL_PREFIX=$(PREFIX) \
+           -DFFMPEG_VER=$(VERSION) \
+           -DENABLE_DAV1D=ON \
+           -DEXTRA_FLAGS="$(FFMPEG_CONFIGURE_OPTIONS)"
```

Comment:
to fix the linux build, https://github.com/xbmc/xbmc/blob/master/tools/depends/configure.ac#L378 needs adjusting